tests/Core/Slider/FluentSliderTests.FluentSlider_Double.verified.razor.html

```diff
@@ -1,2 +1,2 @@
 
-<fluent-slider class="horizontal" min="-1.5" max="7" step="0.5" id="xxx" value="3.5" blazor:onchange="1" blazor:elementreference="xxx"></fluent-slider>
+<fluent-slider class="horizontal" min="-1.5" max="7" step="0.5" id="xxx" value="3,5" blazor:onsliderchange="1" blazor:elementreference="xxx"></fluent-slider>
```

Comment:
Why did you set `3,5` (with a coma)?

Reply:
I did not change that one

Reply:
It is the change you made to the value that is causing this test to fail
`"@BindConverter.FormatValue(Value?.ToString())"` => `@FormatValueAsString(Value)"`

Reply:
Ok, was in (unexplainable) issue in the verified file...